meilisearch-core/src/filters/condition.rs

```diff
@@ -134,14 +159,21 @@ impl<'a> Condition<'a> {
             Some(Value::Number(n)) => { 
                 if let Some(value) = self.value.as_number() {
                     if let Some(cmp_result) = compare_numbers(&n, value) {
-                        return Ok(cmp_result == self.condition)
+                        let res =  match self.condition {
+                            ConditionType::Equal => cmp_result == Ordering::Equal,
+                            ConditionType::GreaterEqual => cmp_result == Ordering::Equal || cmp_result == Ordering::Greater,
+                            ConditionType::LessEqual => cmp_result == Ordering::Equal || cmp_result == Ordering::Less,
+                            ConditionType::Greater=> cmp_result == Ordering::Greater,
+                            ConditionType::Less=> cmp_result == Ordering::Less,
```

Comment:
```suggestion
                            ConditionType::Equal => cmp_result == Ordering::Equal,
                            ConditionType::GreaterEqual => cmp_result == Ordering::Equal || cmp_result == Ordering::Greater,
                            ConditionType::LessEqual => cmp_result == Ordering::Equal || cmp_result == Ordering::Less,
                            ConditionType::Greater => cmp_result == Ordering::Greater,
                            ConditionType::Less => cmp_result == Ordering::Less,
```

Reply:
Would it be possible to change `cmp_result` by a shorter name to make the lines shorter.